I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 238, in script call
    call battle(tonysoprano)
  File "game/script.rpy", line 597, in script
    $ add_or_sub = randint(0,1)
  File "game/script.rpy", line 597, in <module>
    $ add_or_sub = randint(0,1)
NameError: name 'randint' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 238, in script call
    call battle(tonysoprano)
  File "game/script.rpy", line 597, in script
    $ add_or_sub = randint(0,1)
  File "C:\Users\Vile\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Vile\Downloads\renpy-8.1.3-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 597, in <module>
    $ add_or_sub = randint(0,1)
NameError: name 'randint' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
Derleth Manor 1.0
Fri Mar 15 13:01:56 2024
